app/models/candidate_interface/gcse_qualification_details_form.rb

```diff
@@ -0,0 +1,15 @@
+module CandidateInterface
+  class GcseQualificationDetailsForm
+    include ActiveModel::Model
+
+    attr_accessor :grade, :award_year
+    validates :grade, :award_year, presence: true
```

Comment:
I think we can add further validation for `award_year`, you can reuse what was used in degrees.